app/javascript/selectors/portfolioSelector.js

```diff
@@ -0,0 +1,28 @@
+export function getPortfolioName(portfolioId, state) {
+    let portfolios = state.portfolios; 
+    for (let key in portfolios) {
+      if (portfolios[key].id == portfolioId) {
+        return portfolios[key].name
+      }
+    }
+};
+
+export function getSelectedBuildingId(portfolioId, state) {
+    let portfolios = state.portfolios; 
```

Comment:
Same thing here: can rewrite the entire function as
```
portfolios[portfolioId].selected_building
```

(Is there a case where the portfolio doesn't exist and we want to actually return 0? If so please lmk, but it seems safe to assume that the portfolio id is valid and to just skip the loop and if statement and default return value)